lib/seedQuestions/englishTest.js

```diff
@@ -0,0 +1,450 @@
+'use strict';
```

Comment:
A lot of this code seems to be duplicated from the mcq test MD parsing scripts. Refactor this in a way that the code already written is re-used.